nucypher/cli/characters/ursula.py

```diff
@@ -240,10 +240,11 @@ def ursula(click_config,
         click_config.emit(message="WARNING: Running in Federated mode", color='yellow')
 
     # Seed - Step 1
-    teacher_uris = [teacher_uri] if teacher_uri else list()
+    teacher_uris = [teacher_uri] if teacher_uri else None
```

Comment:
``` suggestion
    teacher_uris = [teacher_uri] if teacher_uri else list()
```

This *must* be a list, I think `None` will cause it to fail?

Reply:
As discussed, this is `False`.